spec/spec_helper.rb

```diff
@@ -117,6 +117,10 @@
     printer.print(File.open(filename, 'w'))
   end
 
+  RSpec.configure do |config|
+    config.include ActiveSupport::Testing::TimeHelpers
```

Comment:
Bei anderen Specs haben wir diese Helper nur dort inkludiert, wo sie verwendet wurden. Da das mittlerweile schon 10x der Fall ist, könnte man ab jezt diesen Weg gehen und es immer verfügbar machen. In diesem Fall sollte man aber die anderen `include ActiveSupport::Testing::TimeHelpers` entfernen.